### DISCUSSION TITLE
how to Stream from vMix to MediaMTX via SRT ?

### DISCUSSION THREAD

**Discussion by newsoneisb** [bluenviron/mediamtx#3713] [Questions]
### Question

Hello everyone,

I'm trying to stream from vMix to MediaMTX using SRT, but I'm having some issues. I've followed the steps outlined in the documentation, but I'm still unable to successfully set up the SRT publish via vMix.

In vMix  I Navigated to the "Output/SRT Setting" tab.
Entered the necessary details for the SRT output as follows.

type: caller
hostname: 127.0.0.1
port: 8890
streamid: #!::m=publish,r=mystream

Despite following these steps, I'm unable to establish a connection between vMix and MediaMTX. I've checked my firewall settings and network connectivity, but I haven't been able to resolve the issue.

I would appreciate any assistance or advice on how to troubleshoot this issue.
Thank you!

**Comment by aler9** [comment]
Hello, this is impossible to debug since vMix is a commercial software and you did not provide server logs. If nothing shows in server logs, then it's impossible to understand what the problem is. If something shows up, open a bug report and provide server logs and a network dump.
